src/prefect/cli/task.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+import typer
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.task_server import serve as task_serve
+from prefect.utilities.importtools import import_object
+
+task_app = PrefectTyper(name="task", help="Commands for working with task scheduling.")
```

Comment:
Not sure what to call this other than `task scheduling`.

Reply:
This is good and we can revise it later if we want. 👍